webpack/envs/clientNovoDevelopmentConfig.js

```diff
@@ -13,7 +13,7 @@ export const clientNovoDevelopmentConfig = {
   devtool: "source-map",
   entry: {
     "artsy-novo": [
-      "webpack-hot-middleware/client?name=novo",
+      "webpack-hot-middleware/client?path=http://localhost:3001/__webpack_hmr&name=novo",
```

Comment:
We might want to use a more exotic port since 3001 is so common, just in case users have that set in their `.env` file

Reply:
Totally, I also want to make it configurable.